### PR TITLE
Tech/colocate some stories

### PR DESCRIPTION
**Pull request by tooppaaa**
Merging to #10913 to simplify review

This is only moving stories outside generators and using baseGenerator
Small improvement of generator to handle staticDir

### CODE REVIEW

**Comment by shilman** [comment]
@tooppaaa check out the chromatic diff 🤔

**Comment by tooppaaa** [comment]
Weird

File: lib/cli/src/frameworks/marko/welcome.marko [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+class {
+	onCreate() {}
```
Comment (shilman) [inline lib/cli/src/frameworks/marko/welcome.marko:3]: tabs??

**Review by shilman — APPROVED** [review]
Nice work @tooppaaa! Love to see all that deleted code. One minor comment, plus the weird chromatic thing that we can ignore for now. Feel free to merge whenever you're ready!

**Review by ndelangen — APPROVED** [review]

**Pull request merged**